desc/objectives/linear_objectives.py

```diff
@@ -3238,3 +3238,218 @@ def compute(self, params, constants=None):
 
         """
         return params["Psi"]
+
+
+class FixNearAxis(_FixedObjective):
```

Comment:
Would it make more sense to have separate constraints for R,Z,lambda like we do for the others?